src/main/java/net/geforcemods/securitycraft/gui/GuiSRAT.java

```diff
@@ -0,0 +1,264 @@
+package net.geforcemods.securitycraft.gui;
+
+import java.util.List;
+
+import net.geforcemods.securitycraft.SCContent;
+import net.geforcemods.securitycraft.SecurityCraft;
+import net.geforcemods.securitycraft.containers.ContainerGeneric;
+import net.geforcemods.securitycraft.entity.EntitySentry;
+import net.geforcemods.securitycraft.entity.EntitySentry.EnumSentryMode;
+import net.geforcemods.securitycraft.gui.components.GuiPictureButton;
+import net.geforcemods.securitycraft.network.packets.PacketSUpdateNBTTag;
+import net.geforcemods.securitycraft.network.packets.PacketSetSentryMode;
+import net.geforcemods.securitycraft.util.ClientUtils;
+import net.geforcemods.securitycraft.util.Utils;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.client.gui.inventory.GuiContainer;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.entity.player.InventoryPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraft.util.math.BlockPos;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+public class GuiSRAT extends GuiContainer {
+
+	private static final ResourceLocation TEXTURE = new ResourceLocation(
+			"securitycraft:textures/gui/container/srat.png");
```

Comment:
Should be one the same line, not two lines. Same thing with the field below this one.

Reply:
Eclipse messing around when auto-formatting... Apologies